src/R/Components/Impl/InteractiveWorkflow/Implementation/RSessionCallback.cs

```diff
@@ -90,7 +90,7 @@ internal sealed class RSessionCallback : IRSessionCallback {
 
         public void ViewObject(string expression, string title) {
             var viewer = _coreShell.ExportProvider.GetExportedValue<IObjectViewer>();
-            viewer?.ViewObjectDetails(expression, title);
+            viewer?.ViewObjectDetails(_session, "base::.GlobalEnv", expression, title);
```

Comment:
I'll do a code sweep for this. I added it late in the work, and probably didn't find all places where it needs to go - and there are also some pre-existing ones that will also be better off with an indirection.